adit/core/management/commands/receiver.py

```diff
@@ -87,8 +85,7 @@ async def _send_files(self):
                 # TODO: Maybe store unreadable files in some special folder for later analysis
                 logger.error(
                     f"Error while reading and transmitting received DICOM file '{filename}' "
-                    f"with StudyInstanceUID '{study_uid}', SeriesInstanceUID '{series_uid}', "
-                    f"SOPInstanceUID '{instance_uid}'."
+                    f"with StudyInstanceUID '{study_uid}', SOPInstanceUID '{instance_uid}'."
```

Comment:
We could still include the SeriesInstanceUID in the log, which makes it easier for us to query the DICOM file that led to the problem.